al_aws_collector.js

```diff
@@ -180,10 +180,15 @@ class AlAwsCollector {
                 context.fail(errorString);
             });
         } else {
-            let okStatus = prepareHealthyStatus(streamType);
-            this.sendStatus(okStatus, () => {
+            let okStatus = this.prepareHealthyStatus(streamType);
+            if (moment().minutes() === 0 && moment().seconds() === 0) {
```

Comment:
Not sure how this limits reporting to an hour?

Reply:
For example:  at 1 am ,second and minutes will be zero and then at 2 am this condition will meet again.This condition will be meet only after one hr.

Reply:
Oh I see. I don't think this will work, what if the collector doesn't have an invocation on the hour?

Reply:
e.g. polling interval is typically every 15 minutes, so in cases where it invokes at 00:04, 00:19, 00:34 and 00:49 we'll never go into an ok state right? Or am i missing something?